home/ivp-planning/advanced-tools/dxf-tool.md

```diff
@@ -0,0 +1,80 @@
+# DXF Files
+
+In iVP it is possible to import and export DXF files, that function as a simplified 2d visualisation of 3d models. They can be useful for illustrating a planned layout.
+
+After a DXF file is imported, it will be added to the [hierarchy](../user-interface/the-machine-list.md) and can be [edited](../machines/selecting-and-moving-objects.md) as any other object.
+
+{% hint style="warning" %}
+This is different to an [imported PDF](../getting-started/importing-pdfs.md), which will not be moveable in iVP but can be used as a overlay for the [floor plan](../user-interface/the-floor-plan.md).
+{% endhint %}
+
+{% hint style="info" %}
+For better understanding and training we created a [<img src="../../../.gitbook/assets/YouTube_icon.png" alt="" data-size="line"> YouTube Tutorial](https://www.youtube.com/watch?v=Maxivv824Dk&list=PLlzoGkRUR67houzn5F5ejD3R-kQrDcps5&index=18), where you can find a detailed explanation how to import and export of DXF files.
+{% endhint %}  // Video TBD
+
+## Importing a DXF File:
+
+To import a DXF click **File -> DXF -> Import DXF**. 
+
+![](../../../.gitbook/assets/DXF\_Import\_open.jpg)
+
+A new window will open that allows to navigate to the folder containing the DXF file to be imported. to open the file, simply double click it **or** select it and click Open.
+
+![](../../../.gitbook/assets/DXF\_Import\_file.jpg)
+
+Before iVP Planning imports a DXF file, a prompt will ask to set some parameters that specify how the DXF file will be imported. 
+
+![](../../../.gitbook/assets/DXF\_Import\_settings.jpg)
+
+* _**File path:**_ this line represents the location of the to be imported file, change it by clicking the three dots
+* _**As references:**_ this toggle defines whether or not you reference the dxf file. If checked, you will need to keep the dxf file accessible. (!changing the name or location of that file will break the reference!) 
+* _**Line width:**_ this value represents the width each single line will be drawn (in mm)
+* _**Scale:**_ this value represents the scale **of the DXF to import**, e.g. 1:1 or 16:1
+* _**Unit:**_ this value represents the measurement unit **of the DXF to import**, e.g. mm, cm, inch...
+
+![](../../../.gitbook/assets/DXF\_Import\_comparison.jpg)
+
+{% hint style="info" %}
+Note that the **scale** and the **measurement unit** for imported files refer to the scale/unit the DXF file was planned in and not the scale you want to use in your iVP Planning project. If changed in comparison to the origin; the size of the import might be incorrect.
+{% endhint %}
+
+## Exporting a DXF File:
+
+To export objects from iVP Planning, first select the ojects you want to export.
+
+![](../../../.gitbook/assets/DXF\_Export\_selection.jpg)
+
+Then click **File -> DXF -> Export Selection**.
+
+![](../../../.gitbook/assets/DXF\_Export\_open.jpg)
+
+A new window will open that allows to navigate to the folder the DXF file is to be exported, rename it and save it.
```

Comment:
Wording: I think something like "A new window will open that allows you to choose a location for the exported file" would be easier to read/understand. What do you think?

Reply:
I tried avoiding "you" sentences. changed wording to "A new window will open that allows to choose a location for the exported file"